pkg/kubectl/cmd/resize.go

```diff
@@ -105,8 +105,8 @@ func RunResize(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 	currentSize := cmdutil.GetFlagInt(cmd, "current-replicas")
 	precondition := &kubectl.ResizePrecondition{currentSize, resourceVersion}
 	retry := &kubectl.RetryParams{Interval: retryFrequency, Timeout: retryTimeout}
-
-	if err := resizer.Resize(info.Namespace, info.Name, uint(count), precondition, retry, nil); err != nil {
+	waitForReplicas := &kubectl.RetryParams{kubectl.Interval, kubectl.Timeout}
```

Comment:
nit: it's confusing to have a retryFrequency/Timeout in this file (const on top) but use one in another file, please add a comment and/or change the names so it's clearer.